agent/acl_test.go

```diff
@@ -519,14 +520,15 @@ func TestACL_filterChecks(t *testing.T) {
 	require.False(t, ok)
 }
 
+// TODO: remove?
 func TestACL_ResolveIdentity(t *testing.T) {
 	t.Parallel()
 	a := NewTestACLAgent(t, t.Name(), TestACLConfig(), nil, catalogIdent)
 
 	// this test is meant to ensure we are calling the correct function
 	// which is ResolveTokenToIdentity on the Agent delegate. Our
 	// nil authz resolver will cause it to emit an error if used
-	ident, err := a.resolveIdentityFromToken(nodeROSecret)
+	ident, err := a.delegate.ResolveTokenToIdentity(nodeROSecret)
```

Comment:
Both Client and Server have the same implementation of `ResolveTokenToIdentity`, which is implemented entirely in `ACLResolver` so I'm not sure what this test is testing. Maybe it is no longer relevant? If this is supposed to test the `ACLResolver` method I guess it could be tested without `TestACLAgent`.